### ISSUE TITLE
warning tracking\save_image_patcher.py

### ISSUE THREAD

**Issue by mikinko** [ComfyAssets/ComfyUI_PromptManager#70]
a lot warnings 
system:
pytorch version: 2.8.0+cu129
xformers version: 0.0.32.post2
Enabled fp16 accumulation.
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
Using sage attention
Python version: 3.12.11 | packaged by Anaconda, Inc. | (main, Jun  5 2025, 12:58:53) [MSC v.1929 64 bit (AMD64)]
ComfyUI version: 0.3.65
Initializing frontend: Comfy-Org/ComfyUI_frontend@latest, requesting version details from GitHub...
[Prompt Server] web root: S:\_ComfyUI_env\ComfyUI\web_custom_versions\Comfy-Org_ComfyUI_frontend\1.29.2

warnings:

PromptManager logging initialized
2025-10-17 00:42:21 - promptmanager - INFO - Log level: INFO
Log level: INFO
2025-10-17 00:42:21 - promptmanager - INFO - Log file: S:\_ComfyUI_env\user\default\PromptManager\logs\promptmanager.log
Log file: S:\_ComfyUI_env\user\default\PromptManager\logs\promptmanager.log
2025-10-17 00:42:21 - prompt_manager.file_system - INFO - file_system.py:238 - Found ComfyUI root via user/custom_nodes detection (Windows desktop app): S:\_ComfyUI_env
Found ComfyUI root via user/custom_nodes detection (Windows desktop app): S:\_ComfyUI_env
Initializing tracking system for module: s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager.prompt_manager_tracked
2025-10-17 00:42:21 - promptmanager.tracking.prompt_tracker - INFO - PromptTracker initialized
PromptTracker initialized
2025-10-17 00:42:21 - promptmanager.tracking.save_image_patcher - INFO - Successfully patched SaveImage node
Successfully patched SaveImage node
FETCH ComfyRegistry Data: 5/100
From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.app.run is deprecated. Please use tf.compat.v1.app.run instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.data.experimental.DatasetStructure is deprecated. Please use tf.data.DatasetSpec instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.data.Iterator is deprecated. Please use tf.compat.v1.data.Iterator instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.debugging.assert_equal is deprecated. Please use tf.compat.v1.debugging.assert_equal instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.distribute.get_loss_reduction is deprecated. Please use tf.compat.v1.distribute.get_loss_reduction instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.distributions.FULLY_REPARAMETERIZED is deprecated. Please use tf.compat.v1.distributions.FULLY_REPARAMETERIZED instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.dtypes.as_string is deprecated. Please use tf.strings.as_string instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.errors.error_code_from_exception_type is deprecated. Please use tf.compat.v1.errors.error_code_from_exception_type instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.experimental.output_all_intermediates is deprecated. Please use tf.compat.v1.experimental.output_all_intermediates instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.gfile.Copy is deprecated. Please use tf.io.gfile.copy instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.image.resize_image_with_crop_or_pad is deprecated. Please use tf.image.resize_with_crop_or_pad instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.initializers.global_variables is deprecated. Please use tf.compat.v1.initializers.global_variables instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.io.PaddingFIFOQueue is deprecated. Please use tf.queue.PaddingFIFOQueue instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.layers.InputSpec is deprecated. Please use tf.keras.layers.InputSpec instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.linalg.transpose is deprecated. Please use tf.linalg.matrix_transpose instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.lite.constants.FLOAT is deprecated. Please use tf.float32 instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.lite.TocoConverter is deprecated. Please use tf.compat.v1.lite.TocoConverter instead.

FETCH ComfyRegistry Data: 10/100
From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.logging.DEBUG is deprecated. Please use tf.compat.v1.logging.DEBUG instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.losses.Reduction is deprecated. Please use tf.compat.v1.losses.Reduction instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.manip.batch_to_space_nd is deprecated. Please use tf.batch_to_space instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.metrics.accuracy is deprecated. Please use tf.compat.v1.metrics.accuracy instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.mixed_precision.experimental.LossScale is deprecated. Please use tf.compat.v1.mixed_precision.experimental.LossScale instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.mixed_precision.LossScale is deprecated. Please use tf.compat.v1.mixed_precision.LossScale instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.nn.rnn_cell.BasicLSTMCell is deprecated. Please use tf.compat.v1.nn.rnn_cell.BasicLSTMCell instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.nn.avg_pool is deprecated. Please use tf.nn.avg_pool2d instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.profiler.AdviceProto is deprecated. Please use tf.compat.v1.profiler.AdviceProto instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.python_io.TFRecordCompressionType is deprecated. Please use tf.compat.v1.python_io.TFRecordCompressionType instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.ragged.RaggedTensorValue is deprecated. Please use tf.compat.v1.ragged.RaggedTensorValue instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.random.get_seed is deprecated. Please use tf.compat.v1.random.get_seed instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.resource_loader.get_data_files_path is deprecated. Please use tf.compat.v1.resource_loader.get_data_files_path instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.builder.SavedModelBuilder is deprecated. Please use tf.compat.v1.saved_model.builder.SavedModelBuilder instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.constants.ASSETS_DIRECTORY is deprecated. Please use tf.saved_model.ASSETS_DIRECTORY instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.experimental.save is deprecated. Please use tf.saved_model.save instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.loader.maybe_saved_model_directory is deprecated. Please use tf.saved_model.contains_saved_model instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.signature_constants.CLASSIFY_INPUTS is deprecated. Please use tf.saved_model.CLASSIFY_INPUTS instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.signature_def_utils.MethodNameUpdater is deprecated. Please use tf.compat.v1.saved_model.signature_def_utils.MethodNameUpdater instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.tag_constants.GPU is deprecated. Please use tf.saved_model.GPU instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.saved_model.Builder is deprecated. Please use tf.compat.v1.saved_model.Builder instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.sets.set_difference is deprecated. Please use tf.sets.difference instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.sparse.SparseConditionalAccumulator is deprecated. Please use tf.compat.v1.sparse.SparseConditionalAccumulator instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.spectral.dct is deprecated. Please use tf.signal.dct instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.summary.Event is deprecated. Please use tf.compat.v1.summary.Event instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.test.StubOutForTesting is deprecated. Please use tf.compat.v1.test.StubOutForTesting instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.tpu.experimental.embedding_column is deprecated. Please use tf.compat.v1.tpu.experimental.embedding_column instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.tpu.CrossShardOptimizer is deprecated. Please use tf.compat.v1.tpu.CrossShardOptimizer instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.train.experimental.LossScale is deprecated. Please use tf.compat.v1.train.experimental.LossScale instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.train.AdadeltaOptimizer is deprecated. Please use tf.compat.v1.train.AdadeltaOptimizer instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.

From s:\_ComfyUI_env\custom_nodes\ComfyUI_PromptManager\src\tracking\save_image_patcher.py:599: The name tf.AUTO_REUSE is deprecated. Please use tf.compat.v1.AUTO_REUSE instead.

FETCH ComfyRegistry Data: 15/100
Modular Diffusers is currently an experimental feature under active development. The API is subject to breaking changes in future releases.
2025-10-17 00:42:35 - promptmanager.tracking.save_image_patcher - INFO - Patched 27 generic save_images classes for tracking
Patched 27 generic save_images classes for tracking
PromptManager tracking system initialized

**Comment by mikinko** [comment]
forgot mention beta2 ;)
